src/renderer/components/+extensions/__tests__/extensions.test.tsx

```diff
@@ -103,39 +97,47 @@ describe("Extensions", () => {
 
     fireEvent.click(menuTrigger);
 
-    expect(res.getByText("Disable")).toHaveAttribute("aria-disabled", "false");
-    expect(res.getByText("Uninstall")).toHaveAttribute("aria-disabled", "false");
+    expect(await res.findByText("Disable")).toHaveAttribute("aria-disabled", "false");
```

Comment:
Test should have only one expect.

Reply:
If you like I can revert all of these `get*` -> `find*` changes so that this diff is smaller.